tests/php/Security/MemberTest.php

```diff
@@ -1618,4 +1621,111 @@ public function testChangePasswordToBlankIsValidated()
         $result = $member->changePassword('');
         $this->assertFalse($result->isValid());
     }
+
+    public function testMapInCMSGroupsNoLeftAndMain()
+    {
+        if (class_exists(LeftAndMain::class)) {
+            $this->markTestSkipped('LeftAndMain must not exist for this test.');
+        }
+        $result = Member::mapInCMSGroups();
+        $this->assertInstanceOf(Map::class, $result);
+
+        $this->assertEmpty($result, 'Without LeftAndMain, no groups are CMS groups.');
+    }
+
+    /**
+     * @dataProvider mapInCMSGroupsProvider
```

Comment:
```suggestion
     * @dataProvider provideMapInCMSGroups
```

Reply:
We don't have a standard for this I don't think and I don't see how this is any more correct.... but sure I'll change that.

Reply:
Done